Scripts/Src/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.IO;
+
+namespace BootstrapScript
+{
+    class Program
+    {
+        static int Main(string[] args)
+        {
+            string appName = args[0];
+            Console.WriteLine("Setting up bootstrap for " + appName);
+            string bootstrapRootDir = "";
+            string bootstrapName = "NetCoreBootstrap";
+            foreach (var dir in Directory.GetCurrentDirectory().Split('/'))
+            {
+                bootstrapRootDir += dir + "/";
+                if (dir == appName) break;
+            }
+            Console.WriteLine("Replacing " + bootstrapName + " to " + appName + " in " + bootstrapRootDir);
+            string contents = "";
+            foreach (string file in Directory.EnumerateFiles(bootstrapRootDir, "*", SearchOption.AllDirectories))
+            {
+                if (file.Contains("bootstrap-script")) continue;
+                contents = File.ReadAllText(file);
+                if (contents.Contains(bootstrapName))
+                {
+                    contents = contents.Replace(bootstrapName, appName);
+                    if (file.Contains("/README.md"))
+                    {
+                        contents = contents.Replace("### [Kickoff] Application Setup\n","")
+                                            .Replace("After cloning the bootstrap, run `chmod +x ./Scripts/script.sh`\n","")
+                                            .Replace("Then `./Scripts/script.sh AppName` where `AppName` is your application name.\n","")
+                                            .Replace("Your app is ready. Happy coding!\n","");
+                    }
+                    File.WriteAllText(file, contents);
+                }
+            }   
+            Console.WriteLine("Renaming .csproj ...");
+            File.Move(bootstrapRootDir + bootstrapName + ".csproj", bootstrapRootDir + appName + ".csproj");

```

Comment:
For string concatenation you could use template strings.
As a reference:
https://stackoverflow.com/questions/733378/whats-a-good-way-of-doing-string-templating-in-net